docker/reporter/pom.xml

```diff
@@ -15,8 +16,13 @@
 
```

Comment:
the artifactId of this project still contains `recorder` - change it to `arquillian-cube-docker-reporter`